test/event_store/aggregate_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule EventStore.AggregateTest do
+  use ExUnit.Case
+  import ExUnitProperties
+  import EventStore.Generators
+
+  alias EventStore.Command
+  alias EventStore.SampleAggregate
+
+  setup do
+    Application.put_env(
+      :event_store,
+      :storage_adapter,
+      EventStore.Store.InMemory
+    )
+    EventStore.Store.InMemory.reset
```

Comment:
Does this need to be reset between each test or before all of these tests?

Reply:
between each test due to the occasional duplicate timestamp being generated as the aggregate ID.